modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/ResolverCache.java

```diff
@@ -394,4 +393,8 @@ public Map<String, String> getExternalFileCache() {
     public Map<String, String> getRenameCache() {
         return Collections.unmodifiableMap(renameCache);
     }
+    
+    public ParseOptions getParseOptions() {
+		return parseOptions;
+	}
```

Comment:
Would need to use spaces instead of tabs and correct indentation

Reply:
Done!